lib/matplotlib/axes/_base.py

```diff
@@ -3022,6 +3031,111 @@ def set_ybound(self, lower=None, upper=None):
             else:
                 self.set_ylim(upper, lower, auto=None)
 
+    def get_bound_ylim(self):
+        """
+        Get the y-axis range bound to x-axis range
+        """
+        return self.ylim_bound
+
+    def set_bound_ylim(self, bound=True, **kw):
+        """
+        Bounds the y-axis range to changes of x-axis range
+        Call signature::
+
+          set_bound_ylim(self, *args, **kwargs)
```

Comment:
This call signature is less descriptive than the actual call signature. But in general, I don't think it's necessary anyway.